src/dexcandles-v2.ts

```diff
@@ -15,14 +15,29 @@ export function handleSwapV2(event: SwapV2): void {
   const tokenX = loadToken(Address.fromString(lbPair.tokenX));
   const tokenY = loadToken(Address.fromString(lbPair.tokenY));
 
+  // init token0/token1 to match with V1Pair's tokens order
+  const isSorted = tokenX.id < tokenY.id; // if true, order of tokens matches V1Pair
```

Comment:
Are we sure assembly script ignores case when comparing strings. Is it possible to compare the lowercased addresses?

Reply:
Sounds good.